testsuite/features/core/srv_docker.feature

```diff
@@ -40,7 +40,7 @@ Feature: Prepare server for using Docker
     When I follow the left menu "Images > Stores"
     And I follow "Create"
     And I enter "galaxy-registry" as "label"
-    And I enter "registry.mgr.suse.de" as "uri"
+    And I enter the registry_uri as "uri"
```

Comment:
English, not computese, please :smile_cat: 

`And I enter the registry's URI as "uri"`

Reply:
resolved